### DISCUSSION TITLE
How to connect to a PBI live connection report with cloud based database

### DISCUSSION THREAD

**Discussion by HannesFougner** [DaxStudio/DaxStudio#1010] [Q&A]
Hello,

I have some issues when trying to connect DAX Studio to our database.
As the title states, we are running our datasets on a cloud environment, more specifically Microsofts PBI app.
I am not sure which connection option to use and how to get the necessary details to be able to connect.

Thanks in advance.
//Hannes

**Comment by dgosbell** [comment]
I have all the different connection methods documented here https://daxstudio.org/docs/tutorials/getting-connected/
